staging/src/k8s.io/apiserver/pkg/endpoints/handlers/update.go

```diff
@@ -118,7 +118,8 @@ func UpdateResource(r rest.Updater, scope *RequestScope, admit admission.Interfa
 		trace.Step("Conversion done")
 
 		ae := request.AuditEventFrom(ctx)
-		audit.LogRequestObject(ae, obj, scope.Resource, scope.Subresource, scope.Serializer)
+		objGV := schema.GroupVersion{Group: gvk.Group, Version: gvk.Version}
```

Comment:
construct like this, and move above the `if !scope.AcceptsGroupVersion(gvk.GroupVersion())` line and use objGV as the param to that function as well

```suggestion
		objGV := gvk.GroupVersion()
```